frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -142,6 +142,7 @@ export const saveDashboardAndCards = createThunkAction(
                   }) &&
                   // filter out mappings for deleted series
                   (!card_id ||
+                    action ||
```

Comment:
this was a gnarly bug where it wouldn't save parameter_mappings for action cards because they now have car_ids 😬